docs/content/en/contributing/development/guidelines-pull-request.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Pull Request Guidelines"
+description: "Authelia Development Pull Request Guidelines"
+lead: "This section covers the pull request guidelines."
+date: 2022-05-23T22:03:03+10:00
+draft: false
+images: []
+menu:
+  contributing:
+    parent: "development"
+weight: 232
+toc: true
+---
+
+[Pull Request] guidelines are in place in order to maintain consistency and clearly communicate our process for
+processing merges into the [master] branch.
+
+## Overview
+
+* Ensure the `Allow edits by maintainers` checkbox is checked due to our [Squash Merge](#squash-merge) policy
+* Ensure you avoid a force push due to our [Squash Merge](#squash-merge) policy and [Review](#review) complications
+
+## Squash Merge
+
+Every [Pull Request] will be squash merged into [master]. This requires the [Pull Request] branch to be up-to-date with
+the [master] branch.
+
+## Review
+
+Every [Pull Request] will undergo a formal review process. This process is heavily complicated if you rewrite history
+and/or perform a force push, especially after a maintainer has started a review. As such we request that any action that
```

Comment:
You've got various invalid link references here that just render as `[Pull Request]` for example.

The final sentence also looks incomplete.

---

I didn't see much context regarding what this content may have been previously, and didn't want to wade through the commit history or review discussion to try trace back the original content (if there was any for it).

Just thought I'd bring it to your attention :)